Sources/TokamakCore/Views/Containers/Group.swift

```diff
@@ -19,13 +19,10 @@ public struct Group<Content> {
   }
 }
 
-extension Group: View where Content: View {
-  public var body: Never {
-    neverBody("Group")
-  }
-}
+extension Group: PrimitiveView & View where Content: View {}
```

Comment:
Is there a reason that it's conforming to both PrimitiveView and View? Is the former not a refinement of the latter?

Reply:
Yes, conditionally conforming a type to a "refining" protocol (such as `PrimitiveView`) requires that either that type have separate conditional conformances (i.e. `extension Group: View where Content: View {}` and then the `PrimitiveView` conformance), or that both conformances be explicitly written (as is done above). 

Also, I have no preference as to whether the above is written as `PrimitiveView & View` or `PrimitiveView, View`; whatever you prefer.